### PR TITLE
Add requ. regarding JMeter and minor refinements

### PR DESCRIPTION
**Pull request by lesser1**
- Fix typo
- Add link to JMeter
- Add requ. regarding JMeter readiness
- Strike through open questions that appear irrelevant
- Add references between requ.

### CODE REVIEW

File: docs/dev-manuals/architecture/requirements.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ sidebar_position: 2
...
 
-This document is in the making since we are still in the requirements engineering phase.
+Although we are no longer in the requirements engineering phase, this document may still change in the near future.
 
 :::
 
 ## To-do List and Open Questions RE this document
 
-- [ ] Should we differentiate between the individual stakeholders? Or is it sufficient if we just document their perspective or role (e.g., supervisor, researcher, instead of supervisor XYZ or researcher ABC)? When adding information from another interview with another individual stakeholder, I decided to _try out_ anonymization by naming the stakeholder -- a researcher -- Toni. See the section _Stakeholders._ This is up for discussion of course.
+- [x] Should we differentiate between the individual stakeholders? Or is it sufficient if we just document their perspective or role (e.g., supervisor, researcher, instead of supervisor XYZ or researcher ABC)? When adding information from another interview with another individual stakeholder, I decided to _try out_ anonymization by naming the stakeholder -- a researcher -- Toni. See the section _Stakeholders._ This is up for discussion of course.
```
Comment (nk-coding) [inline docs/dev-manuals/architecture/requirements.md]: if it is done, shouldn't we just remove it from the TODO list?

**Review by nk-coding — APPROVED** [review]

File: docs/dev-manuals/architecture/requirements.md (hunk 1 above)
Comment (lesser1) [reply docs/dev-manuals/architecture/requirements.md]: Idc tbh. I will simply remove it...

File: docs/dev-manuals/architecture/requirements.md [hunk 2]
```diff
@@ -173,7 +174,7 @@ This document is in the making since we are still in the requirements engineerin
 
 ##### Open Questions
 
-- What is klee frontend? The examiner mentioned it.
+- ~~What is klee frontend? The examiner mentioned it.~~
```
Comment (delvh) [inline docs/dev-manuals/architecture/requirements.md]: Why keep the question striked-through? We could also remove it completely.

File: docs/dev-manuals/architecture/requirements.md [hunk 3]
```diff
@@ -124,7 +125,7 @@ This document is in the making since we are still in the requirements engineerin
 
 ##### Open Questions
 
-- Are these models the equivalent of user manuals?
+- ~~Are these models the equivalent of user manuals?~~
```
Comment (delvh) [inline docs/dev-manuals/architecture/requirements.md]: See above.

File: docs/dev-manuals/architecture/requirements.md (hunk 2 above)
Comment (lesser1) [reply docs/dev-manuals/architecture/requirements.md]: Idk, I think it's generally best to not hard delete everything all the time, but I see your point. I will simply remove it.

**Review by legendofa — APPROVED** [review]

**Pull request merged**